client/transactions/declarations.d.ts

```diff
@@ -115,6 +115,8 @@ declare module '@woocommerce/navigation' {
 		type_is_not?: unknown;
 		store_currency_is?: string;
 		search?: string[];
+		status_is?: unknown;
+		status_is_not?: unknown;
```

Comment:
type, status, order should be strings, or?

Reply:
Makes sense as they are predefined. Updated.